pages/index.js

```diff
@@ -15,7 +15,7 @@ export default function Home() {
       let data = {};
 
       if (process.env.NODE_ENV === "development") {
-        data = (await import("../job_stats.json")).default;
+        data = (await import("../data/job_stats.json")).default;
```

Comment:
This needs to remain as is because of the README and .gitignore

```suggestion
        data = (await import("../job_stats.json")).default;
```

Reply:
Actually, looking at this again.

Would it be bad to keep the data in /data and just adjust the readme?
Keeps files more organized, especially if more data/fetch scripts are added later. 
Not a big change.